### DISCUSSION TITLE
Generic Install on OpenShift Container - error:superset.views.base:'NoneType' object has no attribute 'id'

### DISCUSSION THREAD

**Discussion by samwilkinson-git** [apache/superset#19333] [Q&A / Help]
After a successful installation in Docker using the 'pip install apache-superset'; I've tried running the same install but this time in OpenShift.

I have successfully pushed the image into OpenShift, I can access the login page successfully... 

Expected Results:
Should be able to login.

Actual Results:
 **error:superset.views.base:'NoneType' object has no attribute 'id'** Traceback (most recent call last): File "/usr/local/lib/python3.8/site-packages/flask/app.py",
 line 1950, in full_dispatch_request rv = self.dispatch_request() File "/usr/local/lib/python3.8/site-packages/flask/app.py", 
line 1936, in dispatch_request return self.view_functions[rule.endpoint](**req.view_args) File "/usr/local/lib/python3.8/site-packages/flask_appbuilder/security/views.py", 
line 516, in login user = self.appbuilder.sm.auth_user_db( File "/usr/local/lib/python3.8/site-packages/flask_appbuilder/security/manager.py",
 line 874, in auth_user_db self.noop_user_update(first_user) File "/usr/local/lib/python3.8/site-packages/flask_appbuilder/security/sqla/manager.py",
 line 244, in noop_user_update .where(self.user_model.id == user.id)

How to reproduce:
Follow this guide... https://superset.apache.org/docs/installation/installing-superset-from-scratch 
PostGres - Setup
Config - Setup

Any help would be appreciated.
Thanks!

**Comment by srinify** [comment]
Hey there, moving to Discussion so hopefully someone in the community will help with debugging 🙏 If there's a clear bug you can help the community reproduce, then definitely open a Github Issue!
